Sources/Core/iOS/ActivityIndicatorViewObserver.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  ActivityIndicatorViewObserver.swift
+//  Operations
+//
+//  Created by Matthew Holden 16/04/2016.
+//
+//
+
+import UIKit
+
+public protocol ActivityIndicatorViewAnimationInterface {
```

Comment:
Can you make this conform to `class` - as it should be weakly held by the observer.